CRM/Core/Payment/Dummy.php

```diff
@@ -222,8 +222,10 @@ public function doPayment(&$params, $component = 'contribute') {
     // the back-end's canonical set of parameters.  But if a processor
     // does this, it needs to invoke this hook after it has done translation,
     // but before it actually starts talking to its proprietary back-end.
-    if ($propertyBag->getIsRecur()) {
-      $throwAnENoticeIfNotSetAsTheseAreRequired = $propertyBag->getRecurFrequencyInterval() . $propertyBag->getRecurFrequencyUnit();
+    if ($propertyBag->getIsRecur() || !empty($params['contributionRecurID'])) {
+      if (empty($params['frequency_interval']) || empty($params['frequency_unit'])) {
+        CRM_Core_Error::deprecatedWarning('contracted frequency params not passed');
+      }
```

Comment:
Just a note that per https://github.com/civicrm/civicrm-core/pull/28684 I'm not happy with this change and I can't see how it relates to the reported issue since it has nothing to do with installments